statsd/statsd_test.go

```diff
@@ -556,6 +556,31 @@ func TestNilSafe(t *testing.T) {
 	assertNotPanics(t, func() { c.SimpleServiceCheck("", Ok) })
 }
 
+func TestNilError(t *testing.T) {
+	var c *Client
+	tests := []func() error{
+		func() error { return c.SetWriteTimeout(0) },
```

Comment:
nit: This list is missing a few cases like `Incr` and `Decr`. For consistency we should add them all

Reply:
sgtm, I'll add them. I'll also unify this test and the `assertNotPanics`-using ones above, since they should cover essentially the same set of things.